src/App.scss

```
@@ -1 +1,26 @@
-// style go here
+* {
+  box-sizing: border-box;
+  font-family: Arial, sans-serif;
+}
+
+.App {
+  text-align: center;
+  padding: 20px;
+  border-radius: 50px;
+  box-shadow: 0 0 7px 2px rgba($color: #000, $alpha: 0.4);
+}
+
+.label {
+  margin: 0 auto;
+  padding-right: 15px;
```

it is better not to set the dimensions of such a value, but to round up to those values that are divisible by 4 
```suggestion
  padding-right: 16px;
```